src/backend/oracle-data-interface/src/main/java/ca/bc/gov/trafficcourtsonline/oracledatainterface/model/Dispute.java

```diff
@@ -1,65 +1,69 @@
 import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
+import javax.persistence.OneToMany;
 import javax.persistence.Table;
 
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
 //mark class as an Entity   
 @Entity  
 //defining class name as Table name  
-@Table  
+@Table
+@Getter @Setter @NoArgsConstructor
 public class Dispute {
-	@Id  
-	@Column
-	private int disputeId;
-	@Column
-	private String ticketNumber;
-	@Column
-	private String courtLocation;
-	@Column
-	private Date violationDate;
-	@Column
-	private String disputantSurname;
-	@Column
-	private String givenNames;
-	
-	public int getDisputeId() {
-		return disputeId;
-	}
-	public void setDisputeId(int disputeId) {
-		this.disputeId = disputeId;
-	}
-	public String getTicketNumber() {
-		return ticketNumber;
-	}
-	public void setTicketNumber(String ticketNumber) {
-		this.ticketNumber = ticketNumber;
-	}
-	public String getCourtLocation() {
-		return courtLocation;
-	}
-	public void setCourtLocation(String courtLocation) {
-		this.courtLocation = courtLocation;
-	}
-	public Date getViolationDate() {
-		return violationDate;
-	}
-	public void setViolationDate(Date violationDate) {
-		this.violationDate = violationDate;
-	}
-	public String getDisputantSurname() {
-		return disputantSurname;
-	}
-	public void setDisputantSurname(String disputantSurname) {
-		this.disputantSurname = disputantSurname;
-	}
-	public String getGivenNames() {
-		return givenNames;
-	}
-	public void setGivenNames(String givenNames) {
-		this.givenNames = givenNames;
-	}
+	@Id
+	@GeneratedValue
+	@Column
+	@Getter @Setter private int id;
+	@Column
+	@Getter @Setter private String ticketNumber;
+	@Column
+	@Getter @Setter private String courtLocation;
+	@Column
+	@Getter @Setter private Date violationDate;
+	@Column
+	@Getter @Setter private String disputantSurname;
+	@Column
+	@Getter @Setter private String givenNames;
+	@Column
+	@Getter @Setter private String streetAddress;
+	@Column
+	@Getter @Setter private String province;
+	@Column
+	@Getter @Setter private String postalCode;
+	@Column
+	@Getter @Setter private String homePhone;
+	@Column
+	@Getter @Setter private String driversLicense;
+	@Column
+	@Getter @Setter private String driversLicenseProvince;
+	@Column
+	@Getter @Setter private String workPhone;
+	@Column
+	@Getter @Setter private Date dateOfBirth;
+	@Column
+	@Getter @Setter private String enforcementOrganization;
+	@Column
+	@Getter @Setter private Date serviceDate;
+	@OneToMany(mappedBy = "dispute", cascade = CascadeType.ALL)
+	@JsonManagedReference("dispute_ticket_counts")
+	@Getter @Setter private List<TicketCount> ticketCounts;
+    @Column
+    @Getter @Setter private boolean lawyerRepresentation;
+    @Column
+    @Getter @Setter private String interpreterLanguage;
+    @Column
+    @Getter @Setter private boolean witnessIntent;
 }
```

Comment:
I believe Lombok is simpler than this.  The getters and setters are added implicitly.  BTW, you need to have an Eclipse plugin for Lombok to work, otherwise the compiler won't work.   But this works for me. 

I think it's just this:

```
package ca.bc.gov.open.jag.oracledatainterface.model;

import java.sql.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity  
@Getter @Setter @NoArgsConstructor
@Table  
public class Dispute {
	@Id  
	@Column
	private int disputeId;
	@Column
	private String ticketNumber;
	@Column
	private String courtLocation;
	@Column
	private Date violationDate;
	@Column
	private String disputantSurname;
	@Column
	private String givenNames;
	
}

```